backend/serverless.yml

```diff
@@ -42,15 +42,33 @@ resources:
       Properties:
         TableName: appReviewsTable-${self:provider.stage}
         AttributeDefinitions:
-          - AttributeName: id
+          - AttributeName: appIdStore
             AttributeType: S
-          - AttributeName: store
+          - AttributeName: versionDate
             AttributeType: S
+          - AttributeName: compSentiment
+            AttributeType: N
         KeySchema:
-          - AttributeName: id
+          - AttributeName: appIdStore
             KeyType: HASH
-          - AttributeName: store
+          - AttributeName: versionDate
```

Comment:
Still kind of confused about how this works... Are appId and store separate attributes, or are they combined into one attribute? Ideally, we'd have appId and store separately in the DB, but combined they would make up the key.

Reply:
They are combined into one attribute, but only for the key. We can still have separate non-key attributes for appId and store, but the partition key value is both of them combined. I'll look into whether there's a way to automatically combine two attributes into one key, but all the examples seemed to have the combined key as a separate attribute alongside the individual parts' separate attributes.